module-db/Interface/ThreadRecord.cpp

```diff
@@ -203,6 +203,9 @@ std::unique_ptr<db::QueryResult> ThreadRecordInterface::markAsReadQuery(const st
 
     auto response = std::make_unique<db::query::MarkAsReadResult>(ret);
     response->setRequestQuery(query);
+    if (ret) {
+        response->setRecordID(localQuery->id);
```

Comment:
just asking, should the response id be set only if removing was successful? same question for all queries.

Reply:
From Mudita Center perspective only notification after successful operations makes sense.
But this is not a big problem if I'll remove this condition.
In case of notification after an unsuccessful operation, MC just asks about the specific record id and checks its status.

Reply:
I agree in this use case, however I think it would be more logical to return as much info in query result as possible. If we already introduce the ID parameter, we could as well allow binding the result with it each time and process it as needed in e.g. application logic. I suppose it would be more universal and easier to use in the future.

Reply:
Ok, I'll remove the condition

Reply:
Done